avr-specs/sync-from-upstream.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+import subprocess
+import json
+import os
+
+ENCODING = 'utf-8'
+
+
+def main():
+    process = subprocess.Popen(['rustc', '--print', 'target-spec-json', '-Z', 'unstable-options', '--target', 'avr-unknown-gnu-atmega328'],
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.PIPE)
```

Comment:
This only works when `rustc` defaults to nightly.  I'd include a check first, to try whether `rustc` refers to a nightly version.

Reply:
done